Sources/ConsoleKit/Logging/LoggerStyleConfiguration.swift

```diff
@@ -0,0 +1,94 @@
+import Logging
+
+extension Logger {
+    /// A series of parameters describing the `ConsoleStyle`s used by `ConsoleLogger`-backed loggers for rendering
+    /// various aspects of their output. The style configuration does _not_ control which components of a logger's
+    /// output are visible, nor does it describe inclusion or exclusion criteria for particular outputs.
+    ///
+    /// If any given parameter is `nil`, the default fallback is used.
+    public struct StyleConfiguration {
+        /// The style used for all log text with no other defined style. The default is `.plain`
+        let baseStyle: ConsoleStyle
+        
+        /// A dictionary mapping log levels to styles. If an entry exists for a given level, that style _replaces_ the
+        /// base style for log messages with that level. The base style is used for all other levels.
+        let levelStyles: [Logger.Level: ConsoleStyle]
+        
+        /// The style used for a log message's timestamp.
+        let timestampStyle: ConsoleStyle?
+        
+        /// The style used for a log message's label, when visible.
+        let labelStyle: ConsoleStyle?
```

Comment:
`styleConfiguration.labelStyle`. We probably don't need `Style` twice? `logger.configuration.style.timestamp = .info`